product-development/README.md

```diff
@@ -28,9 +28,12 @@ All Tilters are encouraged to collaborate in _definining_ high-level business in
 - Attached stories with relevant ideas, including designs
 - Link to specific metrics and/or charts that this change targets to improve
 
-Victor is responsible for ensuring that the [backlog of unstarted epics](https://app.clubhouse.io/windmill/epics?state_ids=500008016&state_ids=500008001) is continually full and that epics are increasingly being better defined. There should be no shortage of epics. Dan B and Nick, and Victor are responsible for maintaining the priority of the backlog (higher priority toward the top in the Clubhouse view, when the triangle is pointing down).
+## Unstarted epics
+Victor is the owner of all unstarted epics. Victor is responsible for ensuring that the [backlog of unstarted epics](https://app.clubhouse.io/windmill/epics?state_ids=500008016&state_ids=500008001) is continually full and that epics are increasingly being better defined. There should be no shortage of epics. Dan B and Nick, and Victor are responsible for maintaining the priority of the backlog (higher priority toward the top in the Clubhouse view, when the triangle is pointing down).
 
-Anyone advocating for an epic's priority (or priorities in general) should @-mention Victor in an epic and/or raise a discussion in the [weekly epics meeting](#weekly-epics-meeting).
+If you have a question or comment about an unstarted epic's scope, design, timeline, priority, or any other aspect, you should @-mention Victor in the epic for a response. You should [first write an epic comment](../communications#written-communication) before raising a discussion in a meeting.
```

Comment:
Hmmm...

I think there's a legit kind of tension to raise at the epics meeting like 
- "I think this is the wrong problem to be solving, does anyone else agree?"
- "this seems less important than the other things above the line"
- "this seems more important than the things currently above the line"

Epic comments are a poor format for that kind of discussion. The format makes it bad for temperature checks, and also makes it bad when there's external context.

Reply:
(relatedly, I hate how clubhouse doesn't broadcast epic comments to the slack channel, but does broadcast other story comments)

Reply:
> (relatedly, I hate how clubhouse doesn't broadcast epic comments to the slack channel, but does broadcast other story comments)

@nicks :Clubhouse has webhooks for certain events, and I just verified that it will post event data to a webhook url. I could write a server that catches these events and use the slack api to post to a channel. (basically do clubhouse's work for them). https://webhook.site/#!/9a753f96-87c0-4801-8c57-1e0d00ff1863/e3bdc7ee-3d83-412d-bd80-e95bac086673/1

Reply:
```suggestion
If you have a question or comment about an unstarted epic's scope, design, timeline, priority, or any other aspect, you should @-mention Victor in the epic for a response.
```

@nicks : Does this work? (Removing the requirement to first write a comment in an epic.) 

I think this should be in the epic to encourage discussion where everyone can see and participate. But let's not let this preference block this PR (which is that Victor is the owner and people should feel comfortable there is an owner and they can go to Victor for help)

Reply:
> (relatedly, I hate how clubhouse doesn't broadcast epic comments to the slack channel, but does broadcast other story comments)

@nicks : Another partial workaround is having Clubhouse DM you @-mentions in Slack. It's a DM, so other people won't see it. And it's only when somebody @-mentions you.

<img width="1188" alt="Screen Shot 2020-06-03 at 1 22 06 PM" src="https://user-images.githubusercontent.com/10860550/83668327-c2827700-a59d-11ea-92d2-cd70e9f9f30b.png">